processing/src/health_check_easy_ocr.py

```diff
@@ -15,6 +18,8 @@
 # Path to `health_check_files` accord to badgerdoc paths
 # bucket: `post`, path: `files/file_id/file_id.pdf`
 bucket = "post"
+bucket = convert_bucket_name_if_s3prefix(bucket)
```

Comment:
Let's make them distinct: 
```suggestion
_BUCKET= "post"
bucket = convert_bucket_name_if_s3prefix(_BUCKET)
```

Reply:
I would like to keep it as it was, because we could accidentally ruin something, that we are not working on right now